src/sentry/integrations/slack/notify_action.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, *args, **kwargs):
         super(SlackNotifyServiceForm, self).__init__(*args, **kwargs)
 
         self.fields['team'].choices = team_list
+        self.fields['team'].initial = team_list[0][0]
```

Comment:
what if `team_list` is empty?

Reply:
I thought about this but assumed this wouldn't be an issue, since the alert rule won't be available unless an integration is enabled. But that's not entirely true for rules already configured before it was disabled.